__init__.py

```diff
@@ -132,6 +132,22 @@ def remove(self, message):
             self.write_skills_data(skills_data)
             self.speak_dialog('remove.complete', dict(skill=skill.name))
 
+    @intent_file_handler('is.installed.intent')
+    def is_installed(self, message):
+        # Failsafe if padatious matches without skill entity.
+        if not message.data.get('skill'):
+            return self.handle_list_skills(message)
+
+        with self.handle_msm_errors(message.data['skill'], self.remove_word):
+            skill = self.find_skill(message.data['skill'], False)
+
+            if skill.is_local:
+                dialog = 'installed'
+            else:
+                dialog = 'not.installed'
+
+            self.speak_dialog(dialog, dict(skill=skill.name))
```

Comment:
Let's use the "pretty name" that I am adding, so change this to ```dict(skill=self.clean_name(skill))```.  For now that does some manual cleanup of the repo name for pronounceability, soon it will switch to using the friendly name from the README.md and Marketplace card.

Reply:
fantastic! I was thinking of suggesting something like this but wasn't sure how feasible it would be